src/Jackett.Common/Indexers/TorrentLeech.cs

```diff
@@ -19,7 +19,7 @@ namespace Jackett.Common.Indexers
 {
     public class TorrentLeech : BaseWebIndexer
     {
-        public override string[] LegacySiteLinks { get; protected set; } = new string[] {
+        public override string[] LegacySiteLinks { get; protected set; } = {
             "https://v4.torrentleech.org/",
         };
```

Comment:
Array brace should go on its own line to match braces style
```suggestion
        public override string[] LegacySiteLinks { get; protected set; } =
        {
            "https://v4.torrentleech.org/",
        };
```